create_quiz/app/src/main/java/com/appit/sophiemberger/create_quiz/QuizQuestions.java

```diff
@@ -0,0 +1,108 @@
+package com.appit.sophiemberger.create_quiz;
+
+import android.support.annotation.NonNull;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.Queue;
+
+public class QuizQuestions {
+
+    //a queue containing the questions for the quiz (FIFO)
+    //the first question to be created by the user will be displayed first
+   static Queue quizQuestions = new Queue() {
```

Comment:
I see why you're using Queue but using ArrayList would be easier for rendering later with a ListAdapter . ArrayList is ordered by index so you don't have to worry about displaying in sequence. 

Btw, for future reference, you're currently using Queue interface with your own implementation. We should use a Queue class such as with LinkedList, which will save you lots of work :) (https://www.geeksforgeeks.org/queue-interface-java/)